src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3526,4 +3370,135 @@
   <data name="ClearCompleted" xml:space="preserve">
     <value>Clear completed</value>
   </data>
+  <data name="NameWithColon" xml:space="preserve">
+    <value>Name:</value>
+  </data>
+  <data name="StatusCenter_ProcessedSize_Header" xml:space="preserve">
+    <value>{0} of {1} processed</value>
+    <comment>Shown in expanded StatusCenter card item when an file system operation is being held.</comment>
+  </data>
+  <data name="ItemsLower" xml:space="preserve">
+    <value>items</value>
+  </data>
+  <data name="StatusCenter_CompressCanceled_Header" xml:space="preserve">
+    <value>Canceled compressing "{0}" items</value>
```

Comment:
Should "items" be separate so we can switch between singular and plural?

Reply:
yeah let me do that in the upcoming pr. im focusing on the bug fixes.